### PR TITLE
Add grip and make all grip tags consistent

### PR DESCRIPTION
**Pull request by koole**
Adds a 'general' 3x3 grip icon, as opposed to having to pick either horizontal or vertical.

image

I've also made the tags on grip-horizontal equal to grip-vertical and the new grip, as the tags listed apply to all.

### CODE REVIEW

File: icons/grip.svg [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  <circle cx="12" cy="5" r="1" />
+  <circle cx="19" cy="5" r="1" />
+  <circle cx="5" cy="5" r="1" />
+  <circle cx="12" cy="12" r="1" />
+  <circle cx="19" cy="12" r="1" />
+  <circle cx="5" cy="12" r="1" />
+  <circle cx="12" cy="19" r="1" />
+  <circle cx="19" cy="19" r="1" />
+  <circle cx="5" cy="19" r="1" />
+</svg>
```
Comment (ericfennis) [inline icons/grip.svg:21]:
```suggestion
</svg>

```

File: icons/grip.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    "grab",
+    "dots",
+    "handle",
+    "move",
+    "drag"
+  ],
+  "categories": [
+    "layout"
+  ]
+}
```
Comment (ericfennis) [inline icons/grip.json:13]:
```suggestion
}

```

**Review by ericfennis — COMMENTED** [review]
Good work, this is nice to have in the icon set.
We need to add some extra EOF's, but everything else is looking good!

**Comment by koole** [comment]
Great! I’ve added the EOF’s.

**Review by ericfennis — APPROVED** [review]
Nice, good to go!

**Pull request merged**